### PR TITLE
Add API docs for file_urls

### PR DESCRIPTION
**Pull request by devin-civi**
Add API docs for file_urls (in preparation for multiple file uploads.)

This documents both properties for now, using a single file. Once we launch, we'll remove file_key and we'll also change the example to include multiple files.

### CODE REVIEW

File: docs/it-manual/api/program-applications.md [hunk 1]
```diff
@@ -528,6 +528,13 @@ In addition to the metadata field, file upload questions have the following prop
 - **JSON Type**: `string` or `null`
 - **Format**: A URL, `null` if unanswered.
 - **Description**: A link to the file the applicant uploaded.\
+_Note_: This property is deprecated and soon to be replaced by `file_urls`.
+
+#### `file_urls`
+- **Property**: `file_urls`
+- **JSON Type**: `[string]`
+- **Format**: A URL, or an empty array `[]` if unanswered.
```
Comment (avaleske) [inline docs/it-manual/api/program-applications.md:536]: _very_ nit but maybe "An array of URLs, or..." is better?

**Review by avaleske — APPROVED** [review]

File: docs/it-manual/api/program-applications.md [hunk 2]
```diff
@@ -528,6 +528,13 @@ In addition to the metadata field, file upload questions have the following prop
 - **JSON Type**: `string` or `null`
 - **Format**: A URL, `null` if unanswered.
 - **Description**: A link to the file the applicant uploaded.\
+_Note_: This property is deprecated and soon to be replaced by `file_urls`.
```
Comment (avaleske) [inline docs/it-manual/api/program-applications.md:531]: Should we note that this property is removed when the `MULTIPLE_FILE_UPLOAD_ENABLED` flag is turned on?

**Review by avaleske — APPROVED** [review]

File: docs/it-manual/api/program-applications.md (hunk 1 above)
Comment (devin-civi) [reply docs/it-manual/api/program-applications.md:536]: Done

File: docs/it-manual/api/program-applications.md (hunk 2 above)
Comment (devin-civi) [reply docs/it-manual/api/program-applications.md]: Done

**Review by shreyachatterjee00 — APPROVED** [review]

**Pull request merged**